<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Vsevolod/Documents/Snake/Images/Snake-Clipart_13.png</normaloff>C:/Users/Vsevolod/Documents/Snake/Images/Snake-Clipart_13.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Почати гру</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Розмір</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>30</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Перешкоди</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Кількість їжі</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
